src/main/java/de/doubleslash/keeptime/view/ReportController.java

```diff
@@ -92,7 +113,10 @@ private void updateReport(final LocalDate newvalue) {
       for (final Project project : workedProjectsSet) {
          final Label projectName = new Label(project.getName());
          projectName.setFont(labelFontBold);
-         this.gridPane.add(projectName, 0, rowIndex);
+         final Circle circle = new Circle(5, project.getColor());
+
+         this.gridPane.add(circle, 0, rowIndex);
+         this.gridPane.add(projectName, 1, rowIndex);
```

Comment:
please use a HBox for this, instead of adding a new column. so the color circle and project name are in the same column :)

Reply:
good idea, props 👍